docs/get-started/cost-billing/how-pricing-works.md

```diff
@@ -49,6 +54,27 @@ of the base SU for the maximum resource they reserve.
 
     `$46.80`
 
+**OpenShift CPU SU Example:**
+
+- Project with 3 Pods with:
+
+     `1 vCPU, 3 GB RAM, 720hrs (24hr*30days)`
+     `.1 vCPU, 8 GB RAM, 720hrs (24hr*30days)`
```

Comment:
I prefer having leading zeros in decimals less than 1. So, I would write this as "0.1 vCPU".